src/build.less

```diff
@@ -1,3 +1,4 @@
+// TODO: Bootstrap 4 doesn't support LESS. Perhaps we should drop the LESS version for the Bootstrap 4 theme
```

Comment:
My vote is to drop the LESS version completely. Maybe I don't see the forest for the trees, but I doubt there will be many people requiring a custom, LESS-based build of the Select2 theme. Curious what other people think.

Dropping LESS would make the tooling setup for the project much leaner—no need to `diff` the LESS- and Sass-generated versions of the theme CSS anymore. In the past I felt that this step held back a lot of people from contributing (also because it never was set up properly and required manual work which wasn't documented AFAIR).

Reply:
Thanks for taking a look @fk this would be amazing to get in somehow!

Regarding the breaking change, I'd recommend following https://semver.org:
a) major version upgrade: `0.1.0.beta10` becomes `1.0.0`
b) new repo: fork to `select2-bootstrap4-theme` and keep `0.1.0.beta10`
c) add as a separate config instead of overriding the bootstrap3 theme

Either way feel free to drop Less, if you make it a breaking change, here's a 2015 poll about usage:
![sass-vs-less-poll](https://user-images.githubusercontent.com/3094044/36189955-ddfef1f4-1187-11e8-8bb1-65b9dd1f7b47.jpg)

Reply:
@Superpencil just curious about this poll, do you know the source of it?

Reply:
@flo-sch it was just a quick google and I lost the source, however, here's a 2017 survey + source: 
#StateOfJS 2017 Results: CSS & Styling http://stateofjs.com/2017/css/results/